### PR TITLE
Hotfix Dnd bug 🐛 

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/ProfilePage/SmartWalletContents.tsx [hunk 1]
```diff
@@ -45,6 +49,10 @@ const SmartWalletContents = () => {
 
   const dropToSmartWallet = useCallback(
     async (item) => {
+      if (item?.token.type !== CRE8OR) return
+      const owner = await ownerOfERC721(item?.token.tokenId)
+      if (!isMatchAddress(owner, address)) return
```
Comment (sweetmantech) [inline components/ProfilePage/SmartWalletContents.tsx]: remove this check 🐢 

slows down UX

let the smart contract throw an error

Comment (sweetmantech) [reply components/ProfilePage/SmartWalletContents.tsx]:
```suggestion
```

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: components/ProfilePage/SmartWalletContents.tsx (hunk 1 above)
Comment (towardsnewtech) [reply components/ProfilePage/SmartWalletContents.tsx]: fixed! 👍

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**